### ISSUE TITLE
Dead end?

### ISSUE THREAD

**Issue by Hexcles** [tcmg/adventure-issues#107]
**Game URL**
https://developer.chrome.com/devsummit/adventure/#cG9zaXRpb24veC8yODcveS84NA==

**Area**
Outside

**Describe the bug**
Looks like a dead end and I can't reach the swag even though there isn't any visible barrier

**Comment by ChristofferBech** [comment]
Yes I noticed this yesterday too! Will look into why this happens
